devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -14,181 +13,321 @@
-    	standardTile("presence", "device.presence", width: 4, height: 4, canChangeBackground: true) {
-            state "present", label: "Present", labelIcon:"st.presence.tile.present", backgroundColor:"#53a7c0"
-            state "not present", labelIcon:"st.presence.tile.not-present", backgroundColor:"#ffffff"
-        }
-    	standardTile("button", "device.button", decoration: "flat", width: 2, height: 2) {
-        	state "default", icon: "st.unknown.zwave.remote-controller", backgroundColor: "#ffffff"
-        }
-		valueTile("battery", "device.battery", decoration: "flat", width: 2, height: 2) {
-			state "battery", label:'${currentValue}% battery', unit:""
-		}
-
-		main (["presence"])
-		details(["presence","button","battery"])
-	}
+        capability "Refresh"
+        capability "Sensor"
+        capability "Temperature Measurement"
+
+        command "enrollResponse"
+
+        fingerprint inClusters: "0000, 0001, 0003, 0020, 0402, 0B05", outClusters: "0003, 0006, 0008, 0019", manufacturer: "OSRAM", model: "LIGHTIFY Dimming Switch", deviceJoinName: "OSRAM LIGHTIFY Dimming Switch"
+        fingerprint inClusters: "0000, 0001, 0003, 0020, 0500", outClusters: "0003,0019", manufacturer: "CentraLite", model: "3455-L", deviceJoinName: "Iris Care Pendant"
+        fingerprint inClusters: "0000, 0001, 0003, 0007, 0020, 0402, 0B05", outClusters: "0003, 0006, 0019", manufacturer: "CentraLite", model: "3460-L", deviceJoinName: "Iris Smart Button"
+        fingerprint inClusters: "0000, 0001, 0003, 0007, 0020, 0B05", outClusters: "0003, 0006, 0019", manufacturer: "CentraLite", model:"3450-L", deviceJoinName: "Iris KeyFob"
+    }
+
+    simulator {}
+
+    preferences {
+        section {
+            input description: "This feature allows you to correct any temperature variations by selecting an offset. Ex: If your sensor consistently reports a temp that's 5 degrees too warm, you'd enter \"-5\". If 3 degrees too cold, enter \"+3\".", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+            input "tempOffset", "number", title: "Temperature Offset", description: "Adjust temperature by this many degrees", range: "*..*", displayDuringSetup: false
+        }
+        section {
+            input ("holdTime", "number", title: "Minimum time in seconds for a press to count as \"held\"", defaultValue: 1, displayDuringSetup: false)
+        }
+    }
+
+    tiles {
+        standardTile("button", "device.button", width: 2, height: 2) {
+            state "default", label: "", icon: "st.unknown.zwave.remote-controller", backgroundColor: "#ffffff"
+            state "button 1 pushed", label: "pushed #1", icon: "st.unknown.zwave.remote-controller", backgroundColor: "#79b821"
+        }
+
+        valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false) {
+            state "battery", label:'${currentValue}% battery', unit:""
+        }
+
+        valueTile("temperature", "device.temperature", width: 2, height: 2) {
```

Comment:
It doesn't seem optimal to have a temperature tile if 1/2 the supported devices don't support temp. I'll leave this decision up to @tylerlange